program/src/logic.rs

```diff
@@ -105,11 +105,11 @@ pub struct CreateAccountOptions<'a, 'b> {
 /// Create a new account and fund it from the reserve.
 ///
 /// Unlike `system_instruction::create_account`, this will not fail if the account
-/// already exists. This is important, because if account creation would fail for
+/// is already funded. This is important, because if account creation would fail for
```

Comment:
Nit: "if account creation would fail for" might be better as "if account creations fails for"

Reply:
Thanks, applied.